docs/RelayAPI.md

```diff
@@ -806,6 +807,53 @@ The `accuredInterests` is a list with the following elements:
 
 ---
 
+### Information for transfer in transaction
+Returns information about a trustline transfer applied by transaction with given hash.
+#### Request
+```
+GET /transfers/:txHash
+```
+#### Example Request
+```
+curl https://relay0.testnet.trustlines.network/api/v1/transfers/0xC0B33D88C704455075a0724AA167a286da778DDE
+```
+#### URL Parameters
+| Name         | Type                      | Required | Description                                       |
+|--------------|---------------------------|----------|---------------------------------------------------|
+| txHash       | string prefixed with "0x" | YES      | Hash of the tx responsible for the transfer       |
+#### Response
+The response is a an objects with the following elements:
+
+| Attribute        | Type                | JSON Type            | Description                                 |
+| ---------------- | ------------------- | -------------------- | ------------------------------------------- |
+| currencyNetwork  | address             | string               | address of the currency network of transfer |
+| paymentPath      | payment path object | object               | payment path object                         |
+| feesPaid         | list of fees        | array                | list of paid fees along path                |
+
+The `paymentPath` is an object with the following attributes:
+
+| Attribute    | Type              | JSON Type        | Description                                      |
+| ------------ | ----------------- | ---------------- | ------------------------------------------------ |
+| fees         | BigInteger        | string           | total network fees paid for the transfer         |
+| path         | list of addresses | array of strings | Path used for the transfer                       |
+| value        | BigInteger        | string           | value transferred                                |
+| feePayer     | string            | string           | who paid the fees, either `sender` or `receiver` |
+#### Example Response
+```json
+{
+    "currencyNetwork": "0xC0B33D88C704455075a0724AA167a286da778DDE",
+    "paymentPath": {
+        "fees": "1",
```

Comment:
Because PaymentPath would refer to what the sender used as values when making the payment, so it should have everything the sender and receiver agreed on. I can unnest it however, idc.

Reply:
It does not represent the exactly the same. fees are maxFees in the signed data, where here these are the actually used fees.